ddtrace/contrib/pyramid/patch.py

```diff
@@ -16,6 +16,13 @@
 from .trace import trace_pyramid
 
 
+config._add(
```

Comment:
Just a general thought for the future: would it make sense for `_add` to return the newly created configuration and do something like:
~~~ python
int_config = config._add(...)
~~~
and then use `int_config` instead of `config.<int_name>` throughout?

cc @brettlangdon @Kyle-Verhoog @majorgreys

Reply:
👍🏻 yeah, I think that could work

Reply:
Sounds good to me as well, @P403n1x87 do you know if this could be a performance improvement? I vaguely remember you making a similar change with `config` that resulted in a speedup IIRC

Reply:
We're talking fractions of microseconds for each lookup, but the numbers add up if they are done on a hot path

Reply:
I'm fine with it as an ergonomic thing but I'm highly skeptical of any meaningful perf impact 🙂

Reply:
Yep like I said it's just peanuts when compared to other things that can be improved perf-wise. This would be more for saving some typing and potentially increasing readability a wee bit.